NTPClient.h

```diff
@@ -30,6 +31,7 @@ class NTPClient {
     NTPClient(UDP& udp);
     NTPClient(UDP& udp, int timeOffset);
     NTPClient(UDP& udp, const char* poolServerName);
+    NTPClient(UDP& udp, IPAddress poolServerIP);
     NTPClient(UDP& udp, const char* poolServerName, int timeOffset);
     NTPClient(UDP& udp, const char* poolServerName, int timeOffset, int updateInterval);
```

Comment:
Fair enough, I would suggest leaving the current style for now.

We can open another PR later to discuss the default constructor args with @FWeinb.